### ISSUE TITLE
Duplication of the nail bed

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#7727]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 26755636 
F2-IV:2 showing a duplication of the nail bed on digit IV of both hands.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:488232 Split-foot malformation-mesoaxial polydactyly syndrome
 OMIM # 616890 SPLIT-FOOT MALFORMATION WITH MESOAXIAL POLYDACTYLY

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
I am not sure this is ready for an HPO term. The nail bed is defined as skin beneath the nail plate and it is not clear from the figure in the paper that the skin is duplicated as opposed to just being increased. Thoughts?

**Comment by pnrobinson** [comment]
@carolina-f  -- not sure the definition is correct?

**Comment by carolina-f** [comment]
You are right, it doesn't seems very specific

**Comment by pnrobinson** [comment]
I will close this for now, it is hard to know how to define this.
